### ISSUE TITLE
Does it support multitask training, like training color and model in the VehicleID dataset at the same time?

### ISSUE THREAD

**Issue by mrxuehb** [JDAI-CV/fast-reid#474]
As described in the title, the VehicleID dataset has cars' color and model as additional labels. And some commecial products also have both coarse and fine label ids.   So I would like to know does fastreid support mutitask training, whiche I mean training towards multiple targets at the same time?
After reading the dataset code, I saw it generates id label only.

**Comment by lxc86739795** [comment]
Fast-ReID supports multi-task training. Maybe you should modify several modules, e.g., dataloader, prediction head, and loss functions.
